src/components/layout/header/AvatarMenu.tsx

```diff
@@ -64,11 +67,13 @@ export const AvatarMenu = () => {
           <Menu.Item
             icon={<IconLogout size="1rem" />}
             color="red"
-            onClick={() =>
+            onClick={() => {
               signOut({
                 redirect: false,
-              }).then(() => window.location.reload())
-            }
+              }).then(() =>
```

Comment:
Why not make this async and await the Promise? Because it should only be run on success?

Reply:
I don't know, I'm not the one that made it this way. I can try and use promises instead but tbh I find it quite elegant this way and it works without a hitch, for now :x